### PR TITLE
move input to atom, decouple from ListItem

### PR DESCRIPTION
**Pull request by mosch**
Checklist:

- [ ] Test added / Snapshots updated
- [ ] Documentation added / updated

### CODE REVIEW

File: src/atoms/Text.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ if (typeof window !== `undefined`) {
   ])
 }
 
-const availableSizes = {
+export const availableSizes = {
```
Comment (dan-lee) [inline src/atoms/Text.jsx]: Do we need this to be exported?

File: src/atoms/Input.jsx [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+      message: null,
+      length: e.target.value.length,
+    })
+    this.props.onChange && this.props.onChange(e)
+  }
+
+  handleMessageClick = () => this.setState({ message: null })
+
+  render() {
+    const { required, backgroundColor, lines, label, ...props } = this.props
```
Comment (dan-lee) [inline src/atoms/Input.jsx]: If we get rid of this, the tests should pass.

**Comment by dan-lee** [comment]
The gist of this is moving out the input element into `Input` atom component and leaving the `TextInput` basically the same, right?

**Comment by mosch** [comment]
exactly!

**Pull request closed without merging**